dockstore-integration-testing/src/test/resources/dockstore.yml

```diff
@@ -106,6 +106,7 @@ uiConfig:
   dnaStackImportUrl: https://app.dnastack.com/#/app/workflow/import/dockstore
   dnaNexusImportUrl: https://platform.dnanexus.com/panx/tools/import-workflow
   terraImportUrl: https://app.terra.bio/#import-tool/dockstore
+  elwaziImportUrl: https://elwazi.terra.bio/#import-tool/dockstore
```

Comment:
@coverbeck not sure if this is the correct URL, it's composed similarly to the `terraImportUrl` both return a `page not found` if attempting to follow the link.

Also are there configuration env files that I need to update for the various deployments? I was attempting to follow https://github.com/dockstore/dockstore-ui2/blob/e229c2dcdf4e1f61fa2e3e98997bb1ef60dba160/src/app/workflow/launch-third-party/launch-third-party.component.ts#L46

Reply:
Yes, that url is correct

Reply:
You'll also need to update dockstore-deploy, https://github.com/dockstore/dockstore-deploy/blob/94bd66df13a433c4c1bcad5ec2b681160e475fa2/dockstore.yml.template#L160